src/components/DropdownMenu/DropdownMenu.css

```diff
@@ -2,6 +2,19 @@
   margin-right: 8px;
   margin-bottom: 8px;
 }
+
+.dropdown-margin{
+  margin: 20px 30px;
+  font-size: 30px;
+}
+.ant-dropdown-menu-item{
+  font-size: 20px;
+  font-family: monospace;
+  font-weight: 500;
+}
+.ant-drropdown-menu{
```

Comment:
@ritwizsinha Add this padding to menu-item. Rest looks good 👍

Reply:
done